insights/specs/default.py

```diff
@@ -540,6 +540,7 @@ class DefaultSpecs(Specs):
     rpm_ostree_status = simple_command("/usr/bin/rpm-ostree status --json", signum=signal.SIGTERM)
     rpm_pkgs = rpm_pkgs.pkgs_with_writable_dirs
     rsyslog_conf = glob_file(["/etc/rsyslog.conf", "/etc/rsyslog.d/*.conf"])
+    rsyslog_tls_cert_enddate = command_with_args("/usr/bin/openssl x509 -in %s -enddate -noout", ssl_certificate.rsyslog_tls_cert_file)
```

Comment:
Do we need to also add this spec to file `insights/specs/insights_archive.py` ?    @xiangce @huali027

Reply:
@JoySnow I thought `command_with_args ` is just used by "core-collect", so no need to add it. @xiangce Please help do the final confirmation.

Reply:
The `insights_archives.py` is a way (or workaround) for the new core collection to keep compatible with specs that is collected by the legacy collection (aka `uploader.json`)

For new specs that was not added to the `uploader.json`, like `command_with_args` kind specs, it's not required to add its `mangle_name` to the `insights_archives.py`

Reply:
Thanks for the info and confirmation, @huali027, @xiangce .